.claude/scripts/ai-review/prioritizer.py

```
@@ -0,0 +1,299 @@
+            return "M"
+        else:
+            return "S"
+
+    def _print_priority_summary(self) -> None:
+        """우선순위 요약 출력"""
+        critical = sum(1 for i in self.prioritized_issues if i.priority == "Critical")
+        important = sum(1 for i in self.prioritized_issues if i.priority == "Important")
+        suggestion = sum(1 for i in self.prioritized_issues if i.priority == "Suggestion")
+        zero_tolerance = sum(1 for i in self.prioritized_issues if i.zero_tolerance)
+
+        print(f"\n📊 우선순위 분포:")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)